### ISSUE TITLE
ThemeProvider of styled-components doesn't work, while emotion does

### ISSUE THREAD

**Issue by publicJorn** [rebassjs/rebass#798]
Before when I used `@rebass/grid` I could use styled-components' `ThemeProvider` to set the breakpoints and it worked.

Now for `reflexbox/styled-components` it doesn't anymore. However, using the ThemeProvider of emotion does work.

See:  
https://codesandbox.io/s/reflexbox-width-with-styled-components-50pjg

Try swapping around the imports at line 7 & 8, or line 12 & 13.  
You'll see that the combo reflexbox & emotion works and rebass/grid & styled-components also.  
But not reflexbox & styled-components.

My expectation is that reflexbox and styled-components would work.

**Comment by salomvary** [comment]
I've fallen into this trap too, add units to the breakpoints: `breakpoints: ['320px', '768px', '1024px', '1681px']`.

Although the [theme specification](https://github.com/styled-system/styled-system/blob/master/docs/theme-specification.md#breakpoints) suggests using concrete units in many other places Rebass interprets numbers without units as pixels - I think this is rather a bug than a feature ;)

**Comment by publicJorn** [comment]
Right, thanks for pointing that out @salomvary! 
The reflexbox docs do use units, but I got mixed up since I am upgrading from a @rebass/grid project.

I found the cause of the "issue" in the styled-system source: https://github.com/styled-system/styled-system/blob/master/packages/css/src/index.js#L153

A simple check here that converts int values to strings with `px` appended to it would help. Or otherwise throw a warning or something.

I could do a PR, but I'd like to know other peoples opinion about what would be more desirable.

I'd vote for appending `'px'`.
